frontend/src/components/Button/Button.js

```diff
@@ -2,14 +2,15 @@ import React from "react";
 import { TouchableOpacity, Text } from "react-native";
 import { styles } from "./ButtonStyles";
 
-const Button = ({ onPress, title }) => {
+const Button = ({ onPress, title, isPrimary }) => {
```

Comment:
@santosluizfelipe at first glance it looks like the Button issue remains so I have not reviewed the rest of the PR yet. Happy to discuss the issue and why it is an issue on a call but everything in the comment above about Buttons remains. Can I suggest that if everything else is good, you make a branch of this branch to retain the code related to the Button for possible future use in #82 and then remove all Button changes from this branch. I would really like to get this ticket into main.

If I am missing something here, please let me know.

Reply:
@pablisch  I removed the isPrimary from the button, not sure why is showing up here still, could you pull the code and double check because in my local code is sorted.

Reply:
Ah, I see. I was talking about **all** the Button changes but the only reversion I can see is removing the isPrimary prop which actually causes a problem as it is elsewhere and now undefined. I think the work you did may be a good start to the re-usability issue but leaving it as code that is not used anywhere may actually create more work for anyone taking #82 which is what I am trying to avoid in addition to a principle of only passing in functional code into main. 

I have not run the branch yet and need to be elsewhere for a couple of hours. Honestly, I am not concerned what the button looks like at the moment as that will be dealt with when the Button is looked at properly.

Reply:
Hi @santosluizfelipe, I had a chance to run this branch now and I get an error due to the undefined isPrimary mentioned above. Not sure why you do not get the same?

I reverted to the previous Button code and with that error resolved, I'm afraid to say there is still the issue of pressing a required facility does not deselect it but makes it a wanted facility.